src/plugins/intel_cpu/tests/unit/transformations/permute_slice_n_interpolation.cpp

```diff
@@ -0,0 +1,214 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/ov_test_utils.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/interpolate.hpp"
+#include "openvino/op/slice.hpp"
+#include "openvino/op/transpose.hpp"
+#include "transformations/cpu_opset/common/pass/permute_slice_n_interpolation.hpp"
+
+using namespace testing;
+
+class PermuteSliceInterpolateTest: public TransformationTestsF {
+public:
+    PermuteSliceInterpolateTest() {
+        disable_result_friendly_names_check();
+    }
```

Comment:
Why do we need this disabling? Please try to avoid it: all should work after refactoring which was proposed upper

Reply:
Done